outputs.tf

```diff
@@ -11,11 +11,11 @@ output "vpc_id" {
 }
 
 output "public_route_table_id" {
-  value = "${aws_route_table.public.id}"
+  value = "${join(",", aws_route_table.public.*.id)}"
 }
 
 output "private_route_table_id" {
-  value = "${aws_route_table.private.id}"
+  value = "${join(",", aws_route_table.private.*.id)}"
```

Comment:
ok.  i was afraid that changing the type returned by the output (from string to list) would be a backwards-incompatible change; i propose to change the output names from _e.g._ `private_route_table_id` (string) to `private_route_table_ids` (list), is that ok?